src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,73 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+public class AddCommand extends Command {
```

Comment:
Hi, just a thought on naming convention. 
Might this be slightly confusing as a class name due to the use of a verb at the front? After some looking I can see that the class here refers to a command that involves adding of tasks. 

_Perhaps it could even be renamed to CommandToAdd!_